app/Main.hs

```diff
@@ -95,6 +80,11 @@ deployParser = Deploy
             <> help "How many releases to keep, default is '5'"
             )
         )
+  <*> switch
+            ( long "keep-one-failed"
+            <> short 'o'
```

Comment:
Is a short option required? I'm just wondering if `-o` (I think normally short for `--output`) is a good choice.

Reply:
I'll remove it then. If someone asks for it in the future, we can just add it 🙂